public/app/features/datasources/components/picker/DataSourceModal.tsx

```diff
@@ -211,7 +212,11 @@ export function DataSourceModal({
           )}
         </div>
         <div className={styles.newDSSection}>
-          <span className={styles.newDSDescription}>Open a new tab and configure a data source</span>
+          <span className={styles.newDSDescription}>
+            <Trans i18nKey="data-source-picker.configure-new-data-source">
```

Comment:
```suggestion
            <Trans i18nKey="data-source-picker.modal.configure-new-data-source">
```

Reply:
That makes sense 👍,  I've made the changes.